### ISSUE TITLE
Need response

### ISSUE THREAD

**Issue by yuhuixie683** [cyjie429/PMD#2]
In the config.py file

**Comment by roookiiee** [comment]
Excuse, I saw this experiment is based on the SimKGC. So I run SimKGC's codes before this experiment, then I use the model file as the 'eval-model-path', then I can run this code successfully.
I don't konw wheather these operations is right or not?

**Comment by yuhuixie683** [comment]
@roookiiee yes i also got it , inference model just is SimKGC！PMD take SimKGC  as the eval_model when you run the SimKGC and get the its file that was produced after running out

**Comment by yuhuixie683** [comment]
@roookiiee but when you prepare to use yourself original dateset ,you also need to run this original dataset on SimKGC
